lib/hammer_cli_foreman/common_parameter.rb

```diff
@@ -26,6 +26,7 @@ class SetCommand < HammerCLIForeman::Command
 
       option "--name", "NAME", _("parameter name"), :required => true
       option "--value", "VALUE", _("parameter value"), :required => true
+      option "--hidden-value", "HIDDEN_VALUE", _("should the value be hidden")
```

Comment:
While testing I found out one usability issue - the new option should be of type boolean so that the the allowed values are shown in the help and the value is validated.
```ruby
      option "--hidden-value", "HIDDEN_VALUE", _("should the value be hidden"),
        :format => HammerCLI::Options::Normalizers::Bool.new
```